src/bindings/swig/python3/kdb.i

```diff
@@ -38,54 +42,45 @@
 
 // constructors
 %pythonprepend kdb::Key::Key {
-  orig = []
-  if len(args):
-    orig = args[1:]
-    args = [ args[0] ]
+  value = None
+  meta  = {}
+  # check for copy constructor
+  if len(args) and not isinstance(args[0], self.__class__):
+    arg0, args = args[0], args[1:]
+
+    flags = 0
+    args = iter(args)
+    for arg in args:
+      if arg == KEY_END:
+        break
+      elif arg == KEY_SIZE:
+        # ignore value
+        next(args)
+      elif arg == KEY_VALUE:
+        value = next(args)
+      elif arg == KEY_FUNC:
```

Comment:
I mean, isn't it better to not have KEY_FUNC available as long as it isn't supported. A user wouldn't have to fall into the trap when trying to use it ;)

The support of KEY_FUNC in general would be highly appreciated, of course.